### PR TITLE
Update compare-azure-azure-stack.md

### PR DESCRIPTION
**Pull request by jonnychipz**

### CODE REVIEW

**Comment by PRMerger20** [comment]
@jonnychipz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by PatAltimore** [comment]
Hi @jonnychipz,

Thanks for the typo fixes.

Thanks,
Pat

#sign-off

File: azure-stack/operator/compare-azure-azure-stack.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This article describes the differences between global Azure, Azure Stack Hub, an
 
 ## Global Azure
 
-Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It's the freedom to build, manage, and deploy apps on a massive, global network using your favorite tools and frameworks.
+Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It's the freedom to build, manage, and deploy apps on a massive, global network using your favourite tools and frameworks.
```
Comment (sethmanheim) [inline azure-stack/operator/compare-azure-azure-stack.md:30]: This is the British spelling of the word, so the original spelling was correct.

File: azure-stack/operator/compare-azure-azure-stack.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ In addition to the resource providers described above, there are additional PaaS
 
 ### Example use cases for Azure Stack Hub
 
-- Financial modeling
+- Financial modelling
 - Clinical and claims data
```
Comment (sethmanheim) [inline azure-stack/operator/compare-azure-azure-stack.md:59]: This is the British spelling of the word, so the original spelling was correct.

File: azure-stack/operator/compare-azure-azure-stack.md (hunk 1 above)
Comment (PatAltimore) [reply azure-stack/operator/compare-azure-azure-stack.md:30]: Hi @sethmanheim,

I was going to let the commit merge and change it back to simplify the contribution.

Thanks,
Pat

**Comment by PRMerger19** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-stack/operator/compare-azure-azure-stack.md (hunk 1 above)
Comment (ktoliver) [reply azure-stack/operator/compare-azure-azure-stack.md]: I applied the edit/reversal and will merge.

File: azure-stack/operator/compare-azure-azure-stack.md (hunk 2 above)
Comment (ktoliver) [reply azure-stack/operator/compare-azure-azure-stack.md]: I applied the edit/reversal and will merge.

**Comment by ktoliver** [comment]
I applied the edit/reversal and will merge. Thanks, all!

**Pull request merged**